app/src/main/AndroidManifest.xml

```diff
@@ -1,15 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.benoitquenaudon.daggerbundleinjectiondemo">
 
   <application
       android:name=".MyApplication"
       android:allowBackup="true"
+      android:fullBackupContent="@xml/backup_descriptor"
```

Comment:
Android Studio is setup to create it if missing. I can remove it if you'd prefer it. Sorry.